recipe/meta.yaml

```diff
@@ -20,22 +21,23 @@ requirements:
   build:
     - {{ compiler('c') }}
     - make  # [not win]
+    - git  # [win]
     - cmake
 
 test:
   commands:
-    - gif2rgb -h  # [not win]
-    - gifbuild -h  # [not win]
-    - giffix -h  # [not win]
-    - giftext -h  # [not win]
-    - gifclrmp -h  # [not win]
-    - gif2rgb.exe -h  # [win]
-    - gifbuild.exe -h  # [win]
-    - gifclrmp.exe -h  # [win]
-    - gifecho.exe -h  # [win]
-    - giffix.exe -h  # [win]
-    - gifinto.exe -h  # [win]
-    - giftext.exe -h  # [win]
+    # - gif2rgb -h  # [not win]
```

Comment:
Why remove these?

Reply:
I don’t think they’re used and or packaged on Linux distros commonly.

I have them working on windows but it requires to add a copy of getopt.c for one of the exes.

Do you prefer that?

Reply:
Yes, that's fine provided you include the License.

Reply:
Linux distros package them separately. For eg: giflib-tools on Debian